resources/aws/launch_configuration.go

```diff
@@ -22,6 +23,13 @@ type LaunchConfiguration struct {
 	Client *autoscaling.AutoScaling
 }
 
+const (
+	defaultEBSVolumeMountPoint = "/dev/xvdh"
+	// defaultEBSVolumeSize is expressed in GB.
+	defaultEBSVolumeSize = 50
+	defaultEBSVolumeType = "gp2"
+)
```

Comment:
I agree with you. The thing is that the operator should be as stateless as possible and should only reconcile the current state towards the desired state, where all necessary information of the desired state should always come from the TPO.